redbeams/src/main/resources/schema/app/20190530170044_CB-1653_Adjust_database_CRN_storage.sql

```diff
@@ -0,0 +1,22 @@
+-- // CB-1653 Adjust database CRN storage
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE databaseconfig ALTER COLUMN crn TYPE TEXT;
+
+ALTER TABLE databaseconfig ADD COLUMN accountid TEXT;
+
+UPDATE databaseconfig SET accountid = 'unknown' WHERE accountid IS NULL;
```

Comment:
Just a question: could this not go as a default parameter to the previous column?

Reply:
The column ends up as NOT NULL, so having a default column value won't be helpful anyway.